README.md

```diff
@@ -60,73 +60,59 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
```

Comment:
- [ ] Kindly, delete this part as well 😃  for a good look and professional README 👍
 
![dele](https://user-images.githubusercontent.com/57408419/219436778-ac58caaf-d5b8-43c7-ad5e-6c8b01fd1e1c.jpg)